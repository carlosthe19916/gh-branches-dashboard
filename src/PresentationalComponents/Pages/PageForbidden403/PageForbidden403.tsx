import * as React from "react";
import {
  PageSection,
  Bullseye,
  EmptyState,
  EmptyStateVariant,
  Title,
  EmptyStateIcon
} from "@patternfly/react-core";
import { LockIcon } from "@patternfly/react-icons";

export const PageForbidden403: React.FC = ({ children }) => {
  return (
    <React.Fragment>
      <PageSection>
        <Bullseye>
          <EmptyState variant={EmptyStateVariant.full}>
            <EmptyStateIcon icon={LockIcon} />
            <Title headingLevel="h5" size="lg">
              Error 403 Forbidden! The web page (or other resource) that you're
              trying to open in your web browser is a resource that you're not
              allowed to access.
            </Title>
            {children}
          </EmptyState>
        </Bullseye>
      </PageSection>
    </React.Fragment>
  );
};
