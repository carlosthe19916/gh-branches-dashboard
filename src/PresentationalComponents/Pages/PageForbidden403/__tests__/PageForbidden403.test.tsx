import React from "react";
import { shallow } from "enzyme";
import { PageForbidden403 } from "../PageForbidden403";

it("Test snapshot", () => {
  const wrapper = shallow(<PageForbidden403 />);
  expect(wrapper).toMatchSnapshot();
});
