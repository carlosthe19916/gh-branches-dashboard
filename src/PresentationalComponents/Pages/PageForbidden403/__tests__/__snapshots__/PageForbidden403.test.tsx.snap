// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshot 1`] = `
<Fragment>
  <PageSection>
    <Bullseye>
      <EmptyState
        variant="full"
      >
        <EmptyStateIcon
          icon={[Function]}
        />
        <Title
          headingLevel="h5"
          size="lg"
        >
          Error 403 Forbidden! The web page (or other resource) that you're trying to open in your web browser is a resource that you're not allowed to access.
        </Title>
      </EmptyState>
    </Bullseye>
  </PageSection>
</Fragment>
`;
